libass/ass_render.c

```diff
@@ -530,14 +535,14 @@ static bool quantize_transform(double m[3][3], ASS_Vector *pos,
     // D(y_out) <= (D(m_yx) * dx + D(m_yy) * dy) / z0
     //       + 2 * (D(m_zx) * dx + D(m_zy) * dy) * y_lim / z0^2.
 
-    // To estimate acceptable error in matrix coefficient
+    // To estimate acceptable error in matrix coefficient,
     // set error in all other coefficients to zero and solve system
-    // D(x_out) <= ACCURACY & D(y_out) <= ACCURACY for desired D(m_??).
-    // ACCURACY here is some part of total error, i. e. ACCURACY ~ POSITION_PRECISION.
-    // Note that POSITION_PRECISION isn't total error, it's convenient constant.
+    // D(x_out) <= ACCURACY, D(y_out) <= ACCURACY for desired D(m_kl).
+    // Note ACCURACY is of the same magnitude as POSITION_PRECISION (write as: ~)
```

Comment:
English nit: “Note [A is B]” feels weird to me: I’d expect either “Note that A is B” or “Note: A is B” with a colon. Especially with “and that” following it, I think that leaves “Note that” as the only option.

Then again, I’m not actually a native speaker myself (so I may be wrong), and this paragraph has other (pre-existing) slight language problems (such as missing articles), and yet the meaning remains clear, so this isn’t super important.

Reply:
Not sure if it's necessary, but “Note *that* …” should be correct *(but in that case the second *that* seems superfluous)*; changed that and also added articles.